<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAttendance</class>
 <widget class="QMainWindow" name="FaceAttendance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>562</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>562</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>人脸识别打卡系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/bg.jpg);
background-repeat: no-repeat;
background-position: center;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_break">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/images/exitsys.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">/*color: rgb(255, 255, 255);*/</string>
        </property>
        <property name="text">
         <string>人脸识别打卡系统</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_admi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
border-image: url(:/images/admi1.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>618</width>
          <height>427</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">/*color: rgb(255, 255, 255);*/</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>411</width>
           <height>401</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
border:4px solid#2f9ba4;
border-radius:200px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_face">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>60</y>
           <width>300</width>
           <height>300</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
border:2px dotted#63bdc5;
border-radius:150px;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QTextBrowser" name="textBrowserLog">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>281</width>
           <height>411</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">/*background-color:transparent;
color: rgb(255, 255, 255);*/</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_cam">
         <property name="geometry">
          <rect>
           <x>830</x>
           <y>350</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
border-image: url(:/images/cam-off.png);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
